### PR TITLE
chore(ui-pages): add TS types to `ui-pages` package

### PR DESCRIPTION
**Pull request by ke1k0**
Closes: INSTUI-3067

### CODE REVIEW

File: packages/ui-pages/babel.config.js [hunk 1]
```diff
@@ -23,11 +23,6 @@
  */
 
 module.exports = {
-  shouldPrintComment(value) {
-    const isTsRelatedComment = /@ts-expect-error|FIXME/.test(value)
-
-    return !isTsRelatedComment
-  },
```
Comment (HerrTopi) [inline packages/ui-pages/babel.config.js:30]: Should we remove this, since the locator file still has the ts-expect-errors?

**Review by HerrTopi — APPROVED** [review]

File: packages/ui-pages/babel.config.js (hunk 1 above)
Comment (ke1k0) [reply packages/ui-pages/babel.config.js:30]: The Pages component doesn't have locators, so there are no more ts-expect-errors in the package.

File: packages/ui-pages/src/Pages/index.tsx [hunk 2]
```diff
@@ -59,38 +59,39 @@ class Pages extends Component<PagesProps> {
   static defaultProps = {
     children: null,
     defaultPageIndex: null,
```
Comment (matyasf) [inline packages/ui-pages/src/Pages/index.tsx:61]: I think that these 2 are not needed

Comment (ke1k0) [reply packages/ui-pages/src/Pages/index.tsx]: Removed them, thanks!

**Review by matyasf — COMMENTED** [review]
looks good, just one thing: Its Readme.md has no text at all what this component is, just some examples. Could you add 1-2 sentences?

**Review by matyasf — APPROVED** [review]

**Pull request merged**